voice/bxml/callbacks/about.md

```diff
@@ -28,6 +28,7 @@ We use a 15s request timeout when trying to deliver all callbacks, retrying once
 | [Conference Created](conferenceCreated.md)        | Bandwidth API sends this when a new conference is created.                                                                                                                                                   |
 | [Conference Member Join](conferenceMemberJoin.md) | Bandwidth API sends this when a new member joins a conference.                                                                                                                                               |
 | [Conference Member Exit](conferenceMemberExit.md) | Bandwidth API sends this when a member leaves a conference.                                                                                                                                                  |
+| [Conference Redirect](conferenceRedirect.md)      | Bandwidth API sends this to the application when a POST request is made to the [/conferences/{conferenceId}](../../methods/conferences/postConferencesConferenceId.md) endpoint requesting to modify a conference |
```

Comment:
Sadly we have some duplication in the docs, so this list of callbacks also exist in other 2 documents, so you just need to copy this line in `voice/about.md` and in `bxml/about.md` and adjust the link paths accordingly.

Reply:
Also it needs to go in summary so it shows up in the side nav bar

Reply:
thanks! added it to these other pages